refactor(protocol): change nomenclature

maxdesalle opened the pull request:
Replace "money streams" by "token streams", remove the words "salary"...

maxdesalle committed:
refactor(protocol): change nomenclature

PaulRBerg commented on protocol/faq/basics.md:
```diff
@@ -8,8 +8,8 @@ sidebar_position: 1
 
 You can access Sablier through our branded web interfaces:
```
We should instead say "the following web interfaces".

PaulRBerg commented on protocol/faq/basics.md:
```diff
@@ -19,16 +19,16 @@ If you want to use Sablier on a testnet, you may have to get some [TestnetDAI](.
 ### How does streaming work on Sablier?
 
-Imagine a salary worth 3,000 DAI paid by Alice to Bob over the whole month of January.
+Imagine Alice wants to send 3,000 DAI over to Bob during the whole month of January.
```
Suggestion to instead say "Alice wants to make a 3,000 DAI payment to Bob".

PaulRBerg commented on protocol/faq/basics.md:
```diff
@@ -19,16 +19,16 @@ If you want to use Sablier on a testnet, you may have to get some [TestnetDAI](.
-Imagine a salary worth 3,000 DAI paid by Alice to Bob over the whole month of January.
+Imagine Alice wants to send 3,000 DAI over to Bob during the whole month of January.
 
 1. Alice makes the 3,000 DAI deposit in the Sablier contract _before_ Jan 1, setting the stop time to Feb 1.
```
Suggestion to instead say "Alice deposits the 3,000 DAI in Sablier".

PaulRBerg commented on protocol/faq/technical-glossary.md:
```diff
@@ -37,11 +37,11 @@ Sablier is compatible with ERC-20 tokens only, such as [DAI](https://makerdao.co
+| Recipient  | Ethereum address                 |
+| Deposit    | Total amount of tokens to stream |
+| Token      | ERC-20 token address             |
+| Start time | When the token streaming starts  |
```
Suggestion to instead say "The Unix timestamp when the streaming starts".

PaulRBerg commented on protocol/faq/technical-glossary.md:
```diff
@@ -37,11 +37,11 @@ Sablier is compatible with ERC-20 tokens only, such as [DAI](https://makerdao.co
+| Deposit    | Total amount of tokens to stream |
+| Token      | ERC-20 token address             |
+| Start time | When the token streaming starts  |
+| Stop time  | When the token streaming stops   |
```
Suggestion to instead say "The Unix timestamp when the streaming stops".

PaulRBerg commented on protocol/faq/token-streaming.md:
```diff
@@ -22,21 +22,21 @@ In our web interface, the start time is automatically set to ~15 minutes after t
-### Where is the money held?
+### Where are the tokens held?
 
 In our smart contracts. You can verify this assertion by inspecting [Etherscan](https://etherscan.io/address/0xcd18eaa163733da39c232722cbc4e8940b1d8888) or any other blockchain explorer.
```
Can we use the check-summed version of the contract address here?

0xCD18eAa163733Da39c232722cBC4E8940b1D8888

maxdesalle committed:
refactor(protocol): implement requested changes

refactor: replace "our branded web interfaces" with "the following web
interfaces"
refactor: replace "Alice wants to send 3,000 DAI over to Bob" with
"Alice wants to make a 3,000 DAI payment to Bob"
refactor: replace "Alice makes the 3,000 DAI deposit in the Sablier
contract" with "Alice deposits the 3,000 DAI in Sablier"
refactor: replace "When the token streaming starts" with "The Unix
timestamp when the streaming starts"
refactor: replace "When the token streaming stops" with "The Unix timestamp when the streaming stops"

paulrberg commented:
LGTM now, thanks Max!
